README.md

```diff
@@ -1,47 +1,38 @@
-CSS Foldable Display polyfill
+CSS Viewport Segment polyfill
 ===
 
-This is a polyfill for the [proposed](https://github.com/MicrosoftEdge/MSEdgeExplainers/blob/master/Foldables/explainer.md) CSS Foldable Display extensions.
+This is a polyfill for the [proposed](https://drafts.csswg.org/mediaqueries-5/#descdef-media-horizontal-viewport-segments) CSS Viewport Segments.
 
-Web developers targeting foldable devices want to be able to effectively lay out the content in a window that spans multiple displays. CSS Foldable Display
-extensions provides a mean to do that using stylesheets.
+Web developers targeting foldable devices want to be able to effectively lay out the content in a window that spans multiple displays. CSS Viewport Segments provides a mean to do that using stylesheets.
 
-### The 'screen-spanning' CSS media feature
+### The 'horizontal-viewport-segments' and 'horizontal-viewport-segments' CSS media features
 
-The `screen-spanning` CSS media feature can be used to test whether the browser window is spanning across multiple diplays.
+The 'horizontal-viewport-segments' and 'horizontal-viewport-segments' CSS media features can be used to test whether the browser window is displayed across multiple segments (in a dual screen across several displays).
```

Comment:
(in a dual screen across several displays).

Hard to understand.

A display can consist of virtual segments (Windows Zones, foldable displays) or simply have segments because the display is made up of multiple physical displays like dual screen devices.